exporters/otlp/otlplog/otlploggrpc/doc.go

```diff
@@ -47,6 +47,7 @@ OTEL_EXPORTER_OTLP_CERTIFICATE, OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE (default: no
 the filepath to the trusted certificate to use when verifying a server's TLS credentials.
 OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE takes precedence over OTEL_EXPORTER_OTLP_CERTIFICATE.
 The configuration can be overridden by [WithTLSCredentials], [WithGRPCConn] options.
+For self-signed certificates, this configuration should be used together with [WithTLSCredentials].
```

Comment:
This is not true. Moreover, given the precious sentence it does not make any sense

> The configuration can be overridden by [WithTLSCredentials]